docs/dev-setup.md

```diff
@@ -0,0 +1,42 @@
+---
+id: dev-setup
+title: Setup Environment
+---
+
+Gotify requires:
+
+- Go 1.11.5
```

Comment:
```suggestion
- Go 1.11+
```
I would say we don't pin to a certain version of go. 1.11+ would be OK, since 1.12 is coming next week so I think it would not be long before we are upgrading again.

Plus, I think we need to have a prepared workflow for upgrading the build environment to a newer go toolchain version, and at least refuse to build plugins for obsolete versions.